server/server.js

```diff
@@ -135,6 +150,15 @@ APP.put("/checkin", async (req, res) => {
   res.status(200).send("Pet's check in status has been changed.");
 });
 
-APP.listen(PORT, () => {
-  console.log(`Server listening on port ${PORT}`);
-});
+// checks whether to use local cert
```

Comment:
There's an implicit requirement here. It might be helpful to make it clear. "Development mode requires a cert in the `cert` directory." 

Alternatively, this `if` could check for the presence of a `cert` certificate directory, and if it's there, use it. To me, this is s a little simpler... I can reason, "oh, if there's a cert directory, it's used to serve the app."

Again, this is  pretty nuanced feedback... what you have it fine.

Reply:
I clarified the comment, just because I'm only using the certs in order to use https in my endpoints in development mode. It could certainly be modified to use http only in dev mode in someone else's environment.